### DISCUSSION TITLE
What are the domains that define the travel industry?

### DISCUSSION THREAD

**Discussion by kinlane** [OAI/sig-travel#3] [General]
I'd like to kick off a discussion about what the domain model for the travel industry should be. Establishing a well defined set of domains we can develop JSON Schema for, and then define an OpenAPI as well as AsyncAPI for each logical business domain.

**Comment by swaldron58** [comment]
Interesting question. 
First we need to discuss what we mean by domains. 
There are two ways to look at travel products and services. I will use the easy example of rail and air. We can say they are completely separate domains or recognize the fact they are 70 to 80% common in their travel retail needs. Both are in effect, tubes with seats. Both do reservations with tickets (there are some big differences in tickets but won't go down that rabbit hole). Both do what is called legs and segments as part of their schedules and hence what defines their product (seats). Now let's look at rail, hotels and cruise lines. All sell rooms. I could go on. 
At Open Travel we focused more on travel functions as well as industry verticals. A station is a station no matter if it has tracks or a runway next to it (many have both). I'd can provide the current library structure.

**Reply by kinlane** [reply]
I would say that this cross domains and models. I would vote to just start throwing a grab bag of words on a list, and then begin grouping them. I would say rail and air are domains, and seats would be a shared model. I would say that rail, hotels, and cruise lines are domains, and a room is a model within each domain. But again, we should start throwing out words, but also gather existing OpenAPI / Swagger, because they inform the domain + model conversation as well.
